src/SDKs/SqlManagement/Management.Sql/Generated/IDatabaseVulnerabilityAssessmentScansOperations.cs

```diff
@@ -109,13 +112,10 @@ public partial interface IDatabaseVulnerabilityAssessmentScansOperations
         /// <exception cref="Microsoft.Rest.Azure.CloudException">
         /// Thrown when the operation returned an invalid status code
         /// </exception>
-        /// <exception cref="Microsoft.Rest.SerializationException">
-        /// Thrown when unable to deserialize the response
-        /// </exception>
         /// <exception cref="Microsoft.Rest.ValidationException">
         /// Thrown when a required parameter is null
         /// </exception>
-        Task<AzureOperationResponse<IPage<VulnerabilityAssessmentScanRecord>>> ListByDatabaseWithHttpMessagesAsync(string resourceGroupName, string serverName, string databaseName, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
@jaredmoo not just the constructor but these changes too

Reply:
It looks like autorest is being weird and shuffling the methods around the file